src/runtime/mod.rs

```diff
@@ -13,7 +13,16 @@ thread_local! {
     pub(crate) static CONTEXT: RuntimeContext = RuntimeContext::new();
 }
 
-/// The Runtime executor
+/// The Runtime Executor
+///
+/// This is the Runtime for `tokio-uring`.
+/// It wraps the default [`Runtime`], along with the uring [`Driver`]. 
+///
+/// # Multithreading 
```

Comment:
Not sure what this header in the comment is supposed to signify. Maybe instead, a line that says this version only supports the current-thread model. But again, I don't think we need too many details comments for users. The crate isn't meant to be production ready and it isn't really intended for folks who don't know what they want from urings. Plus they need to be prepared for big changes from version to version at this point.

Reply:
This comment block still confuses me. It makes sense when it is in Tokio source as they have two runtimes. We don't at this time. Anyway, the pages says it is outdated, so enough here.